core/client/pom.xml

```diff
@@ -188,6 +188,12 @@
           <scope>provided</scope>
         </dependency>
         <dependency>
+          <groupId>com.google.inject</groupId>
+          <artifactId>guice</artifactId>
+          <version>3.0</version>
```

Comment:
do you see a reason against managing all versions (irrespective of what dependency it is) using dependency management?